content/zh-cn/docs/reference/kubectl/introduction.md

```diff
@@ -0,0 +1,146 @@
+---
+title: "Kubectl 介绍"
+content_type: concept
+weight: 1
+---
+<!--
+title: "Introduction To Kubectl"
+content_type: concept
+weight: 1
+-->
+
+<!--
+Kubectl is the Kubernetes cli version of a swiss army knife, and can do many things.
+
+While this Book is focused on using Kubectl to declaratively manage Applications in Kubernetes, it
+also covers other Kubectl functions.
+-->
+Kubectl 是 Kubernetes CLI 版本的瑞士军刀，可以胜任多种多样的任务。
+
+本文主要介绍如何使用 Kubectl 在 Kubernetes 中声明式管理应用，本文还涵盖了一些其他的 Kubectl 功能。
```

Comment:
The spelling of 'Kubectl', 'Resource', 'Config' are not conforming to the website convention. When translating this page, how about fix those problems?

Reply:
Thanks, raised a new PR to modify the en upstream in #46084